tools/platformio-build.py

```diff
@@ -111,7 +111,8 @@ def scons_patched_match_splitext(path, suffixes=None):
         join(FRAMEWORK_DIR, "tools", "sdk", "include"),
         join(FRAMEWORK_DIR, "tools", "sdk", "libc",
              "xtensa-lx106-elf", "include"),
-        join(FRAMEWORK_DIR, "cores", env.BoardConfig().get("build.core"))
+        join(FRAMEWORK_DIR, "cores", env.BoardConfig().get("build.core")),
+        join(FRAMEWORK_DIR, "tools", "xtensa-lx106-elf", "include")
```

Comment:
PIO error about the missing include comes from here. Proper location should be:
```
join(platform.get_package_dir("toolchain-xtensa"), "include")
```
Because the coreasm.h is here, not inside of `tools/` subdir like with Arduino IDE:
```
~/.platformio/packages/toolchain-xtensa/include/xtensa/coreasm.h
```

Reply:
Success! Thanks again!